### PR TITLE
Feature/cplat 2022 11 01

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by zizw123** [comment]
> Hi, @grizzlytheodore, For review efficiency consideration, when creating a new api version, it is required to place API specs of the base version in the first commit, and push new version updates into successive commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version). Or you could onboard [API spec pipeline](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/606/Enable-API-Spec-Pipeline)

Please follow this guide for review to continue, thanks

**Comment by grizzlytheodore** [comment]
@zizw123  I followed the steps. First commit: [add 2022-11-01 folder](https://github.com/Azure/azure-rest-api-specs/pull/21571/commits/bfdbd3968f9d0d7e12358fd960fa67ec0ff00cac)
[bfdbd39](https://github.com/Azure/azure-rest-api-specs/pull/21571/commits/bfdbd3968f9d0d7e12358fd960fa67ec0ff00cac)
[update readme, update version reference](https://github.com/Azure/azure-rest-api-specs/pull/21571/commits/41b36c339a4241d66c456bb017fc9584e8409999)

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json [hunk 1]
```diff
@@ -0,0 +1,1905 @@
...
+        "enable": {
+          "type": "boolean",
+          "description": "Specifies whether the Terminate Scheduled event is enabled or disabled."
+        }
+      }
+    },
+    "OSImageNotificationProfile": {
+      "type": "object",
+      "properties": {
+        "notBeforeTimeout": {
```
Comment (zizw123) [inline specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json:1750]: >notBeforeTimeout [](http://example.com/codeflow?start=9&length=16)

format date-time  ?

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json [hunk 2]
```diff
@@ -0,0 +1,1905 @@
...
+          "type": "string",
+          "description": "Length of time a Virtual Machine being reimaged or having its OS upgraded will have to potentially approve the OS Image Scheduled Event before the event is auto approved (timed out). The configuration is specified in ISO 8601 format, with the value set to 15 minutes (PT15M)"
+        },
+        "enable": {
+          "type": "boolean",
```
Comment (zizw123) [inline specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json:1755]: >boolean [](http://example.com/codeflow?start=19&length=7)

Consider if you can use enum over boolean

**Comment by zizw123** [comment]
> @zizw123 I followed the steps written above and also what was explained by @rkmanda Please let me know if/where I did it wrong.
> 
> First commit: [add 2022-11-01 folder](https://github.com/Azure/azure-rest-api-specs/pull/21571/commits/bfdbd3968f9d0d7e12358fd960fa67ec0ff00cac) [bfdbd39](https://github.com/Azure/azure-rest-api-specs/pull/21571/commits/bfdbd3968f9d0d7e12358fd960fa67ec0ff00cac) Second commit: [update readme, update version reference](https://github.com/Azure/azure-rest-api-specs/pull/21571/commits/41b36c339a4241d66c456bb017fc9584e8409999) [41b36c3](https://github.com/Azure/azure-rest-api-specs/pull/21571/commits/41b36c339a4241d66c456bb017fc9584e8409999)
> 
> and all the following commits are updates for this release

Thanks, not sure why this is being flagged in the PR, added some minor comments, rest looks good

**Comment by grizzlytheodore** [comment]
thank you ! @zizw123

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json (hunk 1 above)
Comment (grizzlytheodore) [reply specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json:1750]: taking in "PT15" PT8 PT6 etc, so keeping it string !

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json (hunk 2 above)
Comment (grizzlytheodore) [reply specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json:1755]: considered

**Comment by grizzlytheodore** [comment]
we will pursue all the modelValidation fixes outside of this release. 

Either Swagger objects (not examples), or API responses will need to be updated, so we need to work with backend teams on this one which will delay this release significantly if we are to wait until they are all resolved.

**Review by ArcturusZhang — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
Approve the model validation issues for now. Author will open new PR to resolve them.
The suppression is adding suppression to api-version pattern in common.json of this RP. This swagger file only contains models therefore it is fine.

**Pull request merged**